cabal.project

```diff
@@ -317,6 +317,14 @@ source-repository-package
   tag: 714ee03a5a786a05fc57ac5d2f1c2edce4660d85
   --sha256: 1qa4mm36xynaf17990ijmzww0ij8hjrc0vw5nas6d0zx6q9hb978
 
+-- Workaround for duplicate modules warning
+-- https://input-output-rnd.slack.com/archives/CG386Q1NE/p1664547089510409?thread_ts=1664541672.103389&cid=CG386Q1NE
```

Comment:
Lets add a link to the issue https://github.com/astanin/moo/pull/11 so that we can remove the workaround once its resolved.

Reply:
issue has been resolved